packages/react-select/src/components/Select/useSelect.tsx

```diff
@@ -1,6 +1,12 @@
 import * as React from 'react';
 import { getPartitionedNativeProps, resolveShorthand } from '@fluentui/react-utilities';
-import type { SelectProps, SelectState } from './Select.types';
+import { ChevronDown16Regular, ChevronDown20Regular, ChevronDown24Regular } from '@fluentui/react-icons';
+import type { SelectProps, SelectSlots, SelectState } from './Select.types';
+
+/**
+ * Array of all shorthand properties listed as the keys of SelectSlots
+ */
+export const selectShorthandProps: (keyof SelectSlots)[] = ['select', 'icon', 'root'];
```

Comment:
I don't think we do this export anymore.  I don't see it in any of the other components now.
If I'm wrong and we do export it, I think it should be selectSlotPropNames (shorthand is not used in the typing names anymore)

Reply:
oops, haha, looks like a merge artifact that I missed